docs/BEMAN_STANDARD.md

```diff
@@ -171,6 +187,20 @@ add_executable(beman.smart_pointer.tests.roundtrip)
 #...
 ```
 
+**[CMAKE.PASSIVE_TARGETS]** REQUIREMENT: External targets must not modify
+compilation flags of dependents.
+
+Therefore, `target_compile_features` (e.g., `cxx_std_20`) must not be used
+because it modifies the compilation environment of dependent targets. Compiler
+support for required features should be determined at CMake configuration time
+using `check_cxx_source_compiles`.
```

Comment:
hmm, if I'm using a sub library would I not want it compiled in a consistent c++ version?

Reply:
Yes, but you don't want the sub-library to start compiling (part) of your code with a new C++ version because it likes C++2c. 
And if your code is on C++20, but the library really does require C++23, early failure is a good outcome overall. This is a case where failure *is* an option.